run_api.py

```diff
@@ -23,5 +24,5 @@
     if len(sys.argv) > 1 and sys.argv[1].lower().startswith('test'):
         doctest.testmod()
     else:
-        debug = not os.getenv('PRODUCTION')
+        debug = not ast.literal_eval(os.getenv('PRODUCTION', False))
```

Comment:
I like that idea, @jmcarp. We've run into this issue before. Can you put in a ticket to do that? We can forget this PR and I'll just use it locally for debugging work until we can make that change, which should be relatively quick.